### DISCUSSION TITLE
Install issue

### DISCUSSION THREAD

**Discussion by Dave5311** [obsproject/obs-studio#9883] [General]
I have an issue I hope someone can resolve. My studio crashed and will not launch. I am trying to uninstall it but PC keep saying it is running somewhere else. I have run task manager and used an uninstaller to no avail and when I try to reinstall it gives me an error.

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
